crypto/account.go

```diff
@@ -187,23 +187,10 @@ type LogicSigAccount struct {
 	SigningKey ed25519.PublicKey `codec:"sigkey"`
 }
 
-// MakeLogicSigAccountEscrow creates a new escrow LogicSigAccount. The address
-// of this account will be a hash of its program.
-// Deprecated: This method is deprecated for not applying basic sanity check over program bytes,
-// use `MakeLogicSigAccountEscrowChecked` instead.
-func MakeLogicSigAccountEscrow(program []byte, args [][]byte) LogicSigAccount {
-	return LogicSigAccount{
-		Lsig: types.LogicSig{
-			Logic: program,
-			Args:  args,
-		},
-	}
-}
-
 // MakeLogicSigAccountEscrowChecked creates a new escrow LogicSigAccount.
 // The address of this account will be a hash of its program.
 func MakeLogicSigAccountEscrowChecked(program []byte, args [][]byte) (LogicSigAccount, error) {
```

Comment:
Would it make sense to rename this to `MakeLogicSigAccountEscrow` at this point, since there'd no longer be two versions?

Reply:
I think it's a good point, especially since we are making breaking changes here in favor of usability anyway - changed in [a0f7c5e](https://github.com/algorand/go-algorand-sdk/pull/450/commits/a0f7c5e2f676b145c0359bb63f2fbbd4214cd4d3)